libs/api/domains/content-search/src/lib/contentSearch.service.ts

```diff
@@ -38,7 +38,7 @@ export class ContentSearchService implements SearcherService {
     const items = body?.hits?.hits.map(this.fixCase)
 
     return {
-      total: items.length,
+      total: body?.hits?.total?.value ?? items.length,
```

Comment:
Smá undarlegt að við getum ekki treyst því að `body?.hits?.total` sé til.
Kemur þetta kannski tómt ef það er villa eða ef ekkert fannst?

Reply:
trust issues 😅
burt séð frá því, þá sá ég lítið að því að hafa items.length sem fallback, ef svarið frá ES myndi ekki skila reitum sem verið er að vitna í.

en eftiráhyggja, þá gæti það mögulega ollið "földum bug" ef eitthvað færi úrskeiðis, i.e. ef engin villa/viðvörun er köstuð við að reyna að vitna í reit sem er verið að reiða á.

Reply:
optional chaining fjarlægt, ásamt fallback á items.length